organizing_hub/mixins.py

```diff
@@ -45,7 +45,7 @@ def has_permission(self):
         permissions = self.get_permission_required()
 
         """Return False for non-approved local group"""
-        if local_group.status != 'approved':
+        if local_group and local_group.status != 'approved':
```

Comment:
Why change? I think `local_group` can never be `None`, otherwise would have thrown exception in get method?

Reply:
When we override the get_local_gorup method if a group is set to none it makes it to here and throws an error. I was unsure how to fix without adding this.